### PR TITLE
Add test cases

### PR DESCRIPTION
**Pull request by Rogersyp**
I added test cases for Spanish deck, and fix a bug in the for loop when generate cards.

### CODE REVIEW

File: src/main/java/models/Game.java [hunk 1]
```diff
@@ -37,7 +37,7 @@ public void buildDeck(char Choice) {
             }
         }
         else if(Choice == 'S'){
-            for (int i = 0; i < 11; i++){
```
Comment (Nburnettt) [inline src/main/java/models/Game.java:40]: Thanks, I slipped up on that!

File: src/test/java/models/testGame.java [hunk 2]
```diff
@@ -23,6 +23,13 @@ public void testGameBuildDeck(){
     }
 
     @Test
+    public void testSpanishGameBuildDect(){
+        Game g = new Game();
+        g.buildDeck('S');
+        assertEquals(40,g.deck.size());
```
Comment (Nburnettt) [inline src/test/java/models/testGame.java:29]: The build Deck Spanish and English could probably both be in the same test, just by creating two separate objects. This one test would ensure that our buildDeck method always works.

**Comment by Nburnettt** [comment]
I added a few comments, check them out!

File: src/test/java/models/testGame.java (hunk 2 above)
Comment (Rogersyp) [inline src/test/java/models/testGame.java:29]: Good call, thanks man!

**Pull request merged**